independent-projects/tools/devtools-common/src/main/resources/codestarts/quarkus-extension/code/integration-tests/java/integration-tests/pom.tpl.qute.xml

```diff
@@ -49,6 +49,9 @@
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-maven-plugin</artifactId>
+                {#if quarkus.version}
+                <version>$\{quarkus.version}</version>
+                {/if}
```

Comment:
I would rather add it in the `pluginManagement` of the parent.

Reply:
@gsmet I am not sure as the `quarkus-maven-plugin` is only used by the `integration-tests`. Is there a case where we use it in runtime or deployment?

Reply:
Yes but you really don't know what the user will do and better keep it consistent. Having it in `pluginManagement` won't execute it, it will just make it the proper version is used everywhere it is declared.